CSS/documentation.css

```diff
@@ -119,14 +144,37 @@ main{
     font-size: 1.6rem;
 }
 
+/************************** avatar component documentation styles ********************/
 
+.component-display{
+    /* border : 3px solid red; */
+    display : flex;
+    flex-direction: row;
+    align-items : center;
+    justify-content : center;
+    flex-wrap: wrap;
+    gap : 2rem;
+    padding : 1.5rem;
+    background-color : rgba(102, 51, 153, 0.356);
+    border-top-left-radius: 15px;
+    border-top-right-radius: 15px;
+}
 
+.component-code-section{
+    /* border : 3px solid red; */
+    padding : 8px;
+    border-bottom-left-radius: 15px;
+    border-bottom-right-radius: 15px;
+    background-color :  var(--light-test-color);
+}
 
+.code-snippet{
+    border-radius : 15px;
+}
 
-
```

Comment:
provide consistent indentation, telling you this because in future while working somewhere all of these little things are noticed.